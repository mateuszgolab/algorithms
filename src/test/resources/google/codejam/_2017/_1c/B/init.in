11
1 1
540 600
840 900
2 0
900 1260
180 540
1 1
1439 1440
0 1
2 0
0 100
100 200
1 0
100 200
0 1
100 200
2 0
1251 1257
1257 1440
0 2
207 734
58 207
1 1
1 2
2 1
1 1
4 5
2 4
2 0
100 200
300 400
