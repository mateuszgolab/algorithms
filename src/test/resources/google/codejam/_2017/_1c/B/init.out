Case #1: 2
Case #2: 4
Case #3: 2
Case #4: 2
Case #5: 2
Case #6: 2
Case #7: 2
Case #8: 2
Case #9: 2
Case #10: 2
Case #11: 2
